submit.go

```diff
@@ -93,20 +102,24 @@ func main() {
 		splitFile := strings.Split(args.File, "/")
 		protocol := strings.TrimSuffix(splitFile[0], ":")
 		containerName := splitFile[1]
-		storage, err := upload.New(conf[protocol], protocol)
-		if err != nil {
-			log.Fatalf("Error while Initialise Upload Storage: %s", err)
-		}
-
-		if storage == nil {
-			log.Fatalf("No configuration found for protocol %s", protocol)
-		}
-		filesList := strings.Split(args.Upload, ",")
-		for _, file := range filesList {
-			err = storage.Upload(file, containerName)
+		if inTheList(protocol, protocols) {
+			storage, err := upload.New(conf[protocol], protocol)
 			if err != nil {
-				log.Fatalf("Error while uploading file(s): %s", err)
+				log.Fatalf("Error while Initialise Upload Storage: %s", err)
 			}
+
+			if storage == nil {
+				log.Fatalf("No configuration found for protocol %s", protocol)
+			}
+			filesList := strings.Split(args.Upload, ",")
+			for _, file := range filesList {
+				err = storage.Upload(file, containerName)
+				if err != nil {
+					log.Fatalf("Error while uploading file(s): %s", err)
+				}
+			}
+		} else {
+			log.Fatalf("Error while Initialise Upload Storage: protocol %s isn't configured in %s or isn't supported", protocol, configPath)
```

Comment:
Error while Initialise Upload Storage: protocol %s isn't configured in %s or isn't supported
-> 
Error while initializing upload storage: protocol %s isn't configured in %s or isn't supported